dex/msgjson/types.go

```diff
@@ -1080,19 +1259,41 @@ type FeeAsset struct {
 	Amt   uint64 `json:"amount"`
 }
 
+// BondAsset describes an asset for which fidelity bonds are supported.
+type BondAsset struct {
+	Version uint16 `json:"version"` // latest version supported
+	ID      uint32 `json:"id"`
+	Confs   uint32 `json:"confs"`
+	Amt     uint64 `json:"amount"` // to be implied by bond version?
+}
+
 // ConfigResult is the successful result for the ConfigRoute.
 type ConfigResult struct {
+	// APIVersion is the server's communications API version, but we may
+	// consider APIVersions []uint16, with versioned routes e.g. "initV2".
+	// APIVersions []uint16 `json:"apivers"`
+	APIVersion       uint16    `json:"apiver"`
+	DEXPubKey        dex.Bytes `json:"pubkey"`
 	CancelMax        float64   `json:"cancelmax"`
 	BroadcastTimeout uint64    `json:"btimeout"`
-	RegFeeConfirms   uint16    `json:"regfeeconfirms"` // DEPRECATED
 	Assets           []*Asset  `json:"assets"`
 	Markets          []*Market `json:"markets"`
-	Fee              uint64    `json:"fee"` // DEPRECATED
-	APIVersion       uint16    `json:"apiver"`
 	BinSizes         []string  `json:"binSizes"` // Just apidata.BinSizes for now.
-	DEXPubKey        Bytes     `json:"pubkey"`
 
-	RegFees map[string]*FeeAsset `json:"regFees"`
+	BondAssets map[string]*BondAsset `json:"bondAssets"`
+	// BondExpiry defines the duration of time remaining until lockTime below
+	// which a bond is considered expired. As such, bonds should be created with
+	// a considerably longer lockTime. NOTE: BondExpiry in the config response
+	// is temporary, removed when APIVersion reaches BondAPIVersion and we have
+	// codified the expiries for each network (main,test,sim). Until then, the
+	// value will be considered variable, and we will communicate to the clients
+	// what we expect at any given time. BondAsset.Amt may also become implied
+	// by bond version.
+	BondExpiry uint64 `json:"DEV_bondExpiry"`
```

Comment:
Please note this BondExpiry comment.  My feeling is we can settle on a value by network consensus, perhaps implied by the API version or something, but that's not practical while we figure this out. Testnet bond specs will very likely change frequently prior to release.